src/commands/init.cr

```diff
@@ -7,11 +7,17 @@ module Mint
 
       define_argument name,
         description: "The name of the new project",
-        required: true
+        required: false
```

Comment:
I'd suggest leaving this as is since otherwise it might be too easy to initialize a project by accident in the wrong directory. Passing `.` is way more expressive.

Reply:
I can't seem to figure out how to handle if no `name` is passed.
https://github.com/jwaldrip/admiral.cr/

Reply:
We should change it to `false` and handle it manually by raising an error if the name is not present.